src/analyzers/dingo/checks/helper/get_configureMethods.go

```diff
@@ -0,0 +1,45 @@
+package helper
```

Comment:
Please don't mix camel case and snake case. File names should all be in snake case.